src/main/java/org/tikv/common/exception/KeyException.java

```diff
@@ -31,7 +31,7 @@ public KeyException(String errMsg) {
   }
 
   public KeyException(Kvrpcpb.KeyError keyErr) {
-    super("Key exception occurred");
+    super("Key exception occurred " + keyErr.toString());
```

Comment:
Why do you need to print error msg here? 
Have you tried `getKeyErr()`?  it should return the error and you should be able to get the error msg from `getKeyErr().toString()`